provider/k8s/process.go

```diff
@@ -150,6 +159,21 @@ func (p *Provider) ProcessList(app string, opts structs.ProcessListOptions) (str
 		pss = append(pss, *ps)
 	}
 
+	// ignore error because this api only exists if metrics crds are installed in the k8s
+	ms, err := p.MetricsClient.MetricsV1beta1().PodMetricses(p.AppNamespace(app)).List(am.ListOptions{LabelSelector: strings.Join(filters, ",")})
+	if err == nil {
+		metricsByPod := map[string]metricsv1beta1.PodMetrics{}
+		for _, m := range ms.Items {
+			metricsByPod[m.Name] = m
+		}
+
+		for i := range pss {
+			if m, has := metricsByPod[pss[i].Id]; has && len(m.Containers) > 0 {
```

Comment:
just checking. process.Id here is equal the pod name, right?

Reply:
yes, in the existing code I saw using pod name used as id. ref: https://github.com/convox/convox/blob/master/provider/k8s/process.go#L591